meteor/client/lib/editAttribute/edit-attribute-dropdown.tsx

```diff
@@ -106,10 +113,30 @@ const WrappedEditAttributeDropdown = wrapEditAttribute(
 			return !selectedIsAnAvailableOption ? [selectedOption] : []
 		}
 
-		render() {
+		private getAvailableOptionsUsingAlternativeValueIfNecessary(): DropdownOption[] {
+			const currentlySelectedOption = this.getCurrentlySelectedOption()
 			const availableOptions = this.getAvailableOptions()
+
+			const matchOnValue = availableOptions.some((option) => option.value === currentlySelectedOption.value)
+			const matchOnAlternativeValue = availableOptions.some(
```

Comment:
To indicate that these are boolean values, you could rename them from `matchOnX` to `matchedOnX` or `isMatchOnX`.